src/envs/cover.py

```diff
@@ -1019,26 +1014,21 @@ def _Pick_policy(
         desired_x = absolute_params[7]
         desired_y = by + 1e-3
         at_desired_x = abs(desired_x - x) < 1e-5
-        at_desired_y = abs(desired_y - y) < 1e-5
 
         lb, ub = CFG.cover_multistep_action_limits
-        # If we're already above the object and prepared to pick,
-        # then execute the pick (turn up the magnet).
-        if at_desired_x and at_desired_y:
-            return Action(np.array([0., 0., 1.0], dtype=np.float32))
-        # If we're above the object but not yet close enough, move down.
+        # If we're above the object, move down and turn on the gripper.
         if at_desired_x:
             delta_y = np.clip(desired_y - y, lb, ub)
-            return Action(np.array([0., delta_y, -1.0], dtype=np.float32))
```

Comment:
why are all these -1/1's being flipped in the last dimension of the action? did you reverse the meaning of the dgrip dimension?

Reply:
no reversal, I just changed it because I figured it out would be easier to imitate the pick policy if this dimension is constant for the entire duration of picking, and i realized it doesn't otherwise change the result because you won't actually pick until you're close enough

Reply:
(before, the expert would wait until it was in position to turn the gripping on)